src/specBuilder/specUtils.ts

```diff
@@ -296,4 +296,21 @@ export const usePreviousChartData = <T>(data: T) => {
 	}, [data]);
   
 	return previousDataRef.current;
-};
\ No newline at end of file
+};
+
+export const getAnimationMarks = (dimension: string, metric: string, data?: ChartData[], previousData?: ChartData[]) => {
+	let markUpdate = {
+		scale: "yLinear",
+		signal: `datum.${metric} * timerValue`
+	};
+	if (data && previousData) {
+		const isCongruent = data.every((d) => previousData.some((pd) => d[dimension] === pd[dimension]));
+		if (isCongruent) {
```

Comment:
It would be good to put a comment describing what happens in the false condition.  I assume we ignore animations in that case?